### DISCUSSION TITLE
https://auth.tesla.com/oauth2/v3/nts/token 404 error

### DISCUSSION THREAD

**Discussion by ciumagcatalin** [teslamate-org/teslamate#4221] [Q&A]
Hello,

I am trying to authenticate using the Tesla Fleet API,  Third Party Token,  I just created on Tesla Developer for my personal project. The problem is that when I  try to use the  login token and refresh token on :4000 page, it fails authenticating: Error: Tokens are invalid.
When looking on the docker instance I see that it tries to authenticate against a weburl that does not seems to be right: [error] POST https://auth.tesla.com/oauth2/v3/**nts**/token -> 404 .

Any suggestion on why this might happen? I don't want to use the third party authenticate companies. 

Appreciate your help!

Catalin

**Comment by Gyuseung-Kyeong** [comment]
same as here

**Reply by Gyuseung-Kyeong** [reply]
I've resolved this issue by [this](https://docs.teslamate.org/docs/guides/api#tesla-fleet-api)
